docs/backends/monix.md

```diff
@@ -34,48 +34,6 @@ Host header override is supported in environments running Java 12 onwards, but i
 -Djdk.httpclient.allowRestrictedHeaders=host
 ```
 
-## Using async-http-client
-
-To use, add the following dependency to your project:
-
-```scala
-"com.softwaremill.sttp.client3" %% "async-http-client-backend-monix" % "@VERSION@"
-```
-           
-This backend depends on [async-http-client](https://github.com/AsyncHttpClient/async-http-client), uses [Netty](http://netty.io) behind the scenes.
-
-Next you'll need to define a backend instance as an implicit value. This can be done in two basic ways:
```

Comment:
with the exception that the implicit requirement is out of date, maybe we can move this section (describing the Task/Resource difference) up to the HttpClient one, which is much more succint; and this info might be useful to newcomers

Reply:
Moved to "HttpClient" section.